Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ThemeListener/ThemeListenerPage.xaml.cs

```diff
@@ -39,10 +39,15 @@ private void Listener_ThemeChanged(ThemeListener sender)
 
         private void UpdateThemeState()
         {
-            SystemTheme.Text = Listener.CurrentThemeName;
+            SystemTheme.Text = SampleController.Current.ThemeListener.CurrentThemeName;
```

Comment:
We have a SystemTheme method as well:

https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/bdd1d84dc079de3d640d95a2a963c3e37cbc3ccc/Microsoft.Toolkit.Uwp.SampleApp/Pages/SampleController.xaml.cs#L74

(Not sure why this isn't prefixed with 'Get' like CurrentTheme is, probably something else we should clean-up.)

Reply:
Renamed